_data/spring_2015/afearonemoji.json

```diff
@@ -0,0 +1,4 @@
+{
+  "emoji": "dancer",
+  "introduction": "Developer at 18F by day, dancer by night."
+}
```

Comment:
This file should be named `AlexisFearon.json`...mind re-submitting, but with your own emoji and introduction?